InvenTree/company/serializers.py

```diff
@@ -250,29 +253,31 @@ class Meta:
 
         model = SupplierPart
         fields = [
-            'available',
             'availability_updated',
+            'available',
```

Comment:
Question: Why were these fields reordered? There is no technical reason I know for ordering an array in python.